lib/ansible/module_utils/iosxr.py

```diff
@@ -4,7 +4,7 @@
 # still belong to the author of the module, and may assign their own license
 # to the complete work.
 #
-# Copyright (c) 2015 Peter Sprygada, <psprygada@ansible.com>
```

Comment:
yeah, just chatted with @privateip , agreeds to add a new line with RedHat, thanks